### ISSUE TITLE
Namespaces other than WIKI in root folder not shown

### ISSUE THREAD

**Issue by Ejetoro** [michitux/dokuwiki-plugin-move#67]
I have several folders outside the WIKI folder in the root namespace in my wiki, when i want to use the tree based move plugin i am unable to move it because i cannot see them in the tree, all i can see is some *.txt files and the WIKI folder, nothing else.

**Comment by Ejetoro** [comment]
![tree_move](https://cloud.githubusercontent.com/assets/17726060/13606514/7879f522-e54c-11e5-8445-e82e9f7c1319.png)

**Comment by einhirn** [comment]
Those text files are Pages. There's a difference in Dokuwiki between pages (eg. move single page) and namespaces (eg. move tree). And you shouldn't rename a page called "start" or whatever is defined as the default page name for a namespace in your wiki.

But all this has nothing to do with move plugin.

**Comment by Ejetoro** [comment]
I know the difference between pages and folders and i know i should not rename a page called "start", it has nothing to do with my issue. What i told is that move plugin assume by default that all FOLDERS that you are trying to move are inside a folder named "wiki", in my installation, i had several FOLDERS outside "wiki" folder and i cannot see them to use the move plugin.

Move plugin only search for folders to move inside "wiki" folder?. Not a big deal, i just moved it inside the "wiki" folder in order to use the move plugin. (I started to see folders outside "wiki" folder since i installed language plugin, not sure if it was my fault previously... )

Anyway, you should show differences (color, fonts, bullets... whatever) between pages and folders visually.

Thanking you in advance.

**Comment by micgro42** [comment]
> Anyway, you should show differences (color, fonts, bullets... whatever) between pages and folders visually.

The differences are shown by the default template. It is possible that the template your are using does not differentiate between those different classes on the list items.

**Comment by michitux** [comment]
> Move plugin only search for folders to move inside "wiki" folder?.

No, definitely not. All folders should be shown (and are shown in my local installation). The tree for media files also shows some more folders.

I do not really understand what's happening here. My only guess would be that it might be related to the [hidepages](https://www.dokuwiki.org/config:hidepages) configuration option. I've just pushed a change which lets the move plugin ignore this option as imho it doesn't make sense to hide pages in the tree move interface. However, this didn't hide folders before so this is no proper explanation for this behavior, too.

Is this still relevant for you/can you still reproduce this?

**Comment by splitbrain** [comment]
Could it be that you have some namespaces that contain only media files? Can you attach an `ls -lR data/pages` listing here?
